lighthouse-cli/bin.ts

```diff
@@ -35,10 +35,12 @@ import * as Printer from './printer';
 import * as randomPort from './random-port';
 import {Results} from './types/types';
 const yargs = require('yargs');
-const opn = require('opn');
-const updateNotifier = require('update-notifier');
 const pkg = require('../package.json');
 
+// accept noop modules for these, so the real dependency is optional.
+const opn = require('./shim-modules').opn();
```

Comment:
it's just that for these two it's literally just
```js
if (updateNotifier) {
  updateNotifier({pkg}).notify();
}
```
and
```ts
if (flags.view && opn) {
  opn(outputPath, {wait: false});
}
```
the new thing has logged warnings, which is nice, but it is a lot more code to get that same effect :)